tests/conftest.py

```diff
@@ -3,20 +3,20 @@
 import pytest
 # import shutil
 
-from .test_mongo_core import _test_mongetter
+# from .test_mongo_core import _test_mongetter
```

Comment:
Is all this commenting out meant to disable Mongo tests? Anyway, this has to go.

Reply:
Sure